vars/rollingUpgradePipeline.groovy

```diff
@@ -60,6 +60,9 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('test_config', '')}",
                    description: 'Test configuration file',
                    name: 'test_config')
+            string(defaultValue: "${pipelineParams.get('base_versions', '')}",
+                   description: 'Base version in which the upgrade will start from',
```

Comment:
Can you also add example? This is a list, and it's easy to make mistake.

Reply:
all rolling-upgrade*.jenkinsfile this parameter, and you can check the examples (there are also ones with "auto mode", and ones with manually set base version.

Reply:
anyway, requires looking elsewhere to grab the idea, wasting time. Making mistake here also wastes time.
It's a nitpick, so I don't insist.

Reply:
added